### PR TITLE
post 2.13.0 updates

### PR DESCRIPTION
**Pull request by kvalliyurnatt**
Updating necessary files post 2.13 release

### CODE REVIEW

File: deploy/eck-stack/charts/eck-agent/examples/fleet-agents.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # The following example should only be used in conjunction with the 'eck-fleet-server' Helm Chart,
 # and shows how the Agents can be deployed as a daemonset, and controlled by Fleet Server.
 #
-version: 8.15.0-SNAPSHOT
```
Comment (kvalliyurnatt) [inline deploy/eck-stack/charts/eck-agent/examples/fleet-agents.yaml:4]: confused about these, should these be changed ?

File: deploy/eck-stack/charts/eck-elasticsearch/templates/tests/elasticsearch_test.yaml [hunk 2]
```diff
@@ -207,11 +207,11 @@ tests:
           value: my.regis.try/es:8
   - it: should render image properly
     set:
-      image: my.registry.com/elastic/elasticsearch:8.13.2
+      image: my.registry.com/elastic/elasticsearch:8.15.0-SNAPSHOT
```
Comment (kvalliyurnatt) [inline deploy/eck-stack/charts/eck-elasticsearch/templates/tests/elasticsearch_test.yaml:210]: Is this correct ?

File: deploy/eck-stack/templates/tests/elasticsearch_test.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ templates:
 tests:
   - it: should render quickstart properly
     set:
-      eck-elasticsearch.version: 8.2.0
+      eck-elasticsearch.version: 8.15.0-SNAPSHOT
```
Comment (kvalliyurnatt) [inline deploy/eck-stack/templates/tests/elasticsearch_test.yaml:7]: Seems like this has not been updated in a while, so I think this change makes sense?

File: deploy/eck-stack/charts/eck-elasticsearch/templates/tests/elasticsearch_test.yaml (hunk 2 above)
Comment (pebrc) [reply deploy/eck-stack/charts/eck-elasticsearch/templates/tests/elasticsearch_test.yaml:210]: I think it does not matter

**Review by pebrc — APPROVED** [review]

**Comment by kvalliyurnatt** [comment]
buildkite test this -f p=kind -m s=8.14.0,s=8.15.0-SNAPSHOT

**Pull request merged**